Assets/Scripts/ProjectileManagerScript.cs

```diff
@@ -6,12 +7,29 @@ public class ProjectileManagerScript : MonoBehaviour
 {
     public static ProjectileManagerScript Instance;
 
+    private bool areProjectilesInAction;
+
     void Awake() {
         if (Instance == null) {
             Instance = this;
         }
     }
 
+    void Update() {
+        foreach(Transform projectile in transform) {
+            if (projectile != null) {
+                ProjectileBehaviorScript projectileBehaviorScript = projectile.GetComponent<ProjectileBehaviorScript>();
+                if (projectileBehaviorScript != null) {
+                    this.areProjectilesInAction = projectileBehaviorScript.getIsProjectileMoving();
```

Comment:
should this be ORing all the getIsProjectileMoving values? as it is right now, this will only be looking at the latest projectile. If there are slower/faster projectiles, this value might be a false value

Reply:
fixed to OR all values to get total result, then when everything has stopped moving reset the flag